index.md

```diff
@@ -38,24 +60,24 @@ Topics for workshop submissions include, but are not limited to:
 ## Paper Submission
 [Paper submission](https://easychair.org/conferences/?conf=codai22)
 
+## Registration
 
-# Program
-
-The **CODAI** program will consist of keynotes given by tech-leaders both from industry and academia, 
-and paper presentations.
-
-## Keynotes
-* Academic Keynote: Dr. Tianqui Chen, Assistant Professor, Carnegie Mellon
-* Industrial Keynote: Dr. Kailash Gopalakrishnan, Co-Founder & Executive, Ex-IBM Fellow
+[ESWEEK Registration](https://esweek.org/registration/) $15.75 / $20.00
 
-## Registration
  If you have registered the main conference, you can attend the workshop for free.
- OOtherwise, you can attend the workshop by invitation for free.
- [[Request invitation](https://docs.google.com/forms/d/e/1FAIpQLSdoLdRfJAso36fV7vTgVSJcl_Jwyfp5oykXN7mkz_oJdoDXrA/viewform?usp=pp_url)] 
+ 
+## Author Instructions
+
+- Submit camera-ready via Easychair by October 7<sup>th</sup>, 2022
+  - Use [IEEE Template](https://www.ieee.org/conferences/publishing/templates.html) 
+  - Paper must not exceed a length of 5 pages incl. references
```

Comment:
In the CFP, we had 4 papers excluding references. Is this change to „5 papers incl. references“ on purpose in case the IEEE format would lead to too high pressure on the former variant?

Reply:
Hab jetzt erst den Emailverkehr mit Philipp gesehen. 5 Seiten inkl. references passt.